CONTRIBUTING.md

```diff
@@ -25,6 +25,7 @@ Scan through our [existing issues](https://github.com/slipHQ/run-wasm/issues) to
 #### Prerequisites
 
 Make sure you have the following installed in your development environment:
+
```

Comment:
💄 I think this is leftover from my other PR, which was done through GitHub UI so it didn't get formatted correctly until now.